src/domUpdates.js

```diff
@@ -0,0 +1,136 @@
+      return `${recipe.id}` === recipeId
+    }).calculateRecipeCost() / 100).toFixed(2)}`;
+    recipeViewImage.innerHTML = `
+      <img src="${recipeRepo.recipes.find(recipe => 
+      `${recipe.id}` === recipeId).image}" alt="${recipeRepo.recipes.find(recipe => 
+      `${recipe.id}` === recipeId).name}">`
+    show([recipeView]);
+    hide([homePage, browsePage, cookbook]);
+  },
+
+  updateIngredientsDisplay(ingredientsList, currentRecipe) {
+    ingredientsList.innerHTML = '';
+    currentRecipe.ingredients.forEach(ingredient => {
+      ingredientsList.innerHTML += 
+        `<li>${Math.round(ingredient.quantity.amount * 100) / 100} 
+        ${ingredient.quantity.unit} ${ingredient.name}</li>`
+    })
+  },
+
+  updateDirectionsDisplay(directionsList, currentRecipe) {
+    directionsList.innerHTML = '';
+    currentRecipe.instructions.forEach(direction => {
+      directionsList.innerHTML += `<li>${direction.instruction}</li>`
+    })
+  },
+
+  updateFilteredRecipes(homePage, recipeView, cookbook, browsePage, searchBar, tagInput) {
+    hide([homePage, recipeView, cookbook]);
+    show([browsePage]);
+    searchBar.value = '';
+    tagInput.selectedIndex = 0;
+  },
+
+  updateToggle(recipeId, buttons, color) {
+    buttons.forEach(button => {
+      button.parentNode.id === recipeId ?
+        button.classList.toggle(`${color}`) : null;
+    })
+  },
+
+  updateButtons(list, user, buttons, color, category) {
+    list.forEach(recipe => {
+      user[category].includes(recipe) ? 
+        buttons.forEach(button => {
+          button.parentNode.id === `${recipe.id}` ? 
+            button.classList.add(`${color}`) : null;
+        }) : null;
+    });
+  },
+
+  updateCookbookView(
+    cookbookButton,
+    allRecipesButton,
+    favoritesButton,
+    homePage,
+    recipeView,
+    browsePage,
+    cookbook,
+    searchBar,
+    tagInput
+    ) {
+      cookbookButton.classList.add('grey');
+      allRecipesButton.classList.remove('grey');
+    favoritesButton.classList.remove('grey');
+    hide([homePage, recipeView, browsePage]);
+    show([cookbook]);
+    searchBar.value = '';
+    tagInput.selectedIndex = 0;
+  },
+
+  displayHomeView(cookbookButton, allRecipesButton, favoritesButton, browsePage, recipeView, cookbook, homePage) {
+    cookbookButton.classList.remove('grey');
+    allRecipesButton.classList.remove('grey');
+    favoritesButton.classList.remove('grey');
+    hide([browsePage, recipeView, cookbook]);
+    show([homePage]);
+  },
+
+  updateNavBarButtonColor(cookbookButton, allRecipesButton, favoritesButton, recipeRepo) {
+    cookbookButton.classList.remove('grey');
+    allRecipesButton.classList.remove('grey');
+    favoritesButton.classList.remove('grey');
+    recipeRepo.filterState === 'all' ?
+      allRecipesButton.classList.add('grey') :
+      recipeRepo.filterState === 'favorites' ?
+        favoritesButton.classList.add('grey') : null;
+  }
+}
+
+export default domUpdates;
```

Comment:
Thanks for creating this domUpdates  file so our dom manipulation stays here and gets imported to our scripts file for more dry code